src/Approximations/ballinf_approximation.jl

```diff
@@ -0,0 +1,27 @@
+# ===========================================================
+# Concrete overapproximation with a ball in the infinity norm
+# ===========================================================
+
+"""
+    ballinf_approximation(S::LazySet)
+
+Overapproximate a convex set by a tight ball in the infinity norm.
+
+### Input
+
+- `S` -- convex set
+
+### Output
+
+A tight ball in the infinity norm.
+"""
+function ballinf_approximation(S::LazySet)
+    return overapproximate(S, BallInf)
```

Comment:
i didn't expect the function in the new file to call `overapproximate(S, BallInf)`, i thought you would outsource the code here.

Reply:
Good catch, I just copy-pasted that part. I'll change it as well!